src/module_TOUCHSCREEN/TouchscreenBackEnd.java

```diff
@@ -6,7 +6,7 @@
  * 
  */
 
-package module_TOUCHSCREEN;
```

Comment:
So I initially was using that to try to build my DataPacket so the message could be of variable length, but that added too much complexity that got eliminated one I just moved to a fixed-size packet. I'll remote it with this next commit :)